implementations/micrometer-registry-otlp/src/main/java/io/micrometer/registry/otlp/OtlpConfig.java

```diff
@@ -264,4 +292,14 @@ default TimeUnit baseTimeUnit() {
         return getTimeUnit(this, "baseTimeUnit").orElse(TimeUnit.MILLISECONDS);
     }
 
+    static <V> Map<String, V> propertiesStringToMap(String propertiesString,
```

Comment:
I don't like that this is a public method. We should probably move it somewhere internal instead.

Reply:
Moved to `PropertyValidator` in 04616d8